example_site/src/components/ConfigurationControls.js

```diff
@@ -134,306 +150,348 @@ export default function ConfigurationControls({ isOpen }) {
 
   const formClasses = classnames({
     "configuration-controls-form": true,
-    panel: true,
-    open: isOpen,
-    closed: !isOpen,
+    'modal-fade': true,
```

Comment:
Since this is no longer just being used for modals, let's broaden up the classname, like `fade-in-out` or something like that.